emperor/support_files/js/view-controller.js

```diff
@@ -247,35 +241,15 @@ define([
   EmperorViewController.prototype.constructor = EmperorViewControllerABC;
 
   /**
-   * Retrieves the name of the currently active decomposition view.
    *
-   * @return {String} A key corresponding to the active decomposition view.
-   */
-  EmperorViewController.prototype.getActiveDecompViewKey = function() {
-    return this.activeViewKey;
-  };
-
-  /**
-   * Changes the currently active decomposition view.
+   * Retrieve a view from the controller.
    *
-   * @param {String} k Key corresponding to active decomposition view.
-   * @throws {Error} The key must exist, otherwise an exception will be thrown.
-   */
-  EmperorViewController.prototype.setActiveDecompViewKey = function(k) {
-    if (this.decompViewDict[k] === undefined) {
-      throw new Error('This key does not exist, "' + k + '" in the ' +
-                      'the decompViewDict.');
-    }
-    this.activeViewKey = k;
-  };
-
-  /**
-   * Retrieves the currently active decomposition view.
+   * This class does not operate on single decomposition views, hence this
+   * method retrieves the first available view.
    *
-   * @return {DecompositionView} The currently active decomposition view.
    */
-  EmperorViewController.prototype.getActiveView = function() {
-    return this.decompViewDict[this.getActiveDecompViewKey()];
+  EmperorViewController.prototype.getView = function() {
+    return this.decompViewDict[Object.keys(this.decompViewDict)[0]];
```

Comment:
Remember, this was temporary, since this only grabs the first key.  This will need to be changed for biplots. This doesn't need to be fixed here,  but we should add a todo, or an issue so we don't forget that [0] needs to be replaced with a var